js/utils/DatatableUtils.js

```diff
@@ -54,7 +54,7 @@ define(['services/MomentAPI', 'xss', 'appConfig', 'services/AuthAPI', '../const'
         const getFacetForCreatedBy = getCreatedByLogin;
 
         const getFacetForDesign = d =>
-            d.hasWriteAccess || (d.createdBy && authApi.subject() === d.createdBy.login)
+            d.hasWriteAccess || (d.createdBy && authApi.subject() === d.createdBy.name)
```

Comment:
In case of different login and name we will always get "Other designs" because authApi.subject() contains user login Subject is set in AuthAPI.loadUserInfo